lib/checks/shared/avoid-inline-spacing.js

```diff
@@ -0,0 +1,15 @@
+const inlineSpacingCssProperties = [
+	'line-height',
+	'letter-spacing',
+	'word-spacing'
+];
+
+const hasImportantPriority = inlineSpacingCssProperties.some(
+	property => node.style.getPropertyPriority(property) === `important`
+);
+
+if (hasImportantPriority) {
+	return false; //-> fail
```

Comment:
Is the "fail" comment necessary here? IIRC `return false` is how _all_ checks "fail".